MSAL/src/native_auth/public/state_machine/error/ResendCodeError.swift

```diff
@@ -26,11 +26,16 @@ import Foundation
 
 @objc
 public class ResendCodeError: MSALNativeAuthError {
+    /// Describes an error that provides messages describing why an error occurred and provides more information about the error.
     public override var errorDescription: String? {
         if let description = super.errorDescription {
             return description
         }
 
         return "General error"
     }
+
+    init(message: String? = nil) {
+        super.init(identifier: "Resend code", message: message)
```

Comment:
Shouldn't this string be "General error" because is the only type of error that this error class can has?
Anyway, I agree with Rodhan when he says that we should have a type in each class, even if they have one single type. So, we can be more consistent

Reply:
Done (I've used `general_error` as identifier)